src/config/config.ts

```diff
@@ -9,15 +9,21 @@ convict.addFormat({
 })
 
 convict.addFormat({
-  name: "required-positive-number",
+  name: "required-whole-number",
```

Comment:
Hmm I am abit confused about this change :( 
there is no case where any of these env vars (ISOMERPAGES_REPO_PAGE_COUNT, AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS, MAX_NUM_OTP_ATTEMPTS, OTP_EXPIRY,DB_MIN_POOL and DB_MAX_POOL) to have the value of 0 right? 

if any of the above variables can have a value of 0, does it makes sense to have two schemas instead? one that allows 0 and one that doesnt?

Reply:
actually yea, i wanted to check with you guys if there is any env vars that you think 0 will be a valid option for in the current list
Else i can just change it to required-natural-number

Reply:
after looking through I think 0 does not seem to be a valid option for any of the env vars. will proceed to rename the check and asset for > 0 instead
let me know if any concerns @kishore03109 @seaerchin @alexanderleegs